account/account_test.go

```diff
@@ -89,37 +89,39 @@ func TestTransactionHashInvoke(t *testing.T) {
 	testSet := map[string][]testSetType{
 		"mock": {
 			{
-				// https://goerli.voyager.online/tx/0x73cf79c4bfa0c7a41f473c07e1be5ac25faa7c2fdf9edcbd12c1438f40f13d8
-				ExpectedHash:   utils.TestHexToFelt(t, "0x73cf79c4bfa0c7a41f473c07e1be5ac25faa7c2fdf9edcbd12c1438f40f13d8"),
+				// https://sepolia.voyager.online/tx/0x5d307ad21a407ab6e93754b2fca71dd2d3b28313f6e844a7f3ecc404263a406
+				ExpectedHash:   utils.TestHexToFelt(t, "0x5d307ad21a407ab6e93754b2fca71dd2d3b28313f6e844a7f3ecc404263a406"),
 				SetKS:          true,
-				AccountAddress: utils.TestHexToFelt(t, "0x043784df59268c02b716e20bf77797bd96c68c2f100b2a634e448c35e3ad363e"),
+				AccountAddress: utils.TestHexToFelt(t, "0x06fb2806bc2564827796e0796144f8104581acdcbcd7721615ad376f70baf87d"),
 				PrivKey:        utils.TestHexToFelt(t, "0x043b7fe9d91942c98cd5fd37579bd99ec74f879c4c79d886633eecae9dad35fa"),
-				PubKey:         "0x043784df59268c02b716e20bf77797bd96c68c2f100b2a634e448c35e3ad363e",
-				ChainID:        "SN_GOERLI",
+				PubKey:         "0x06fb2806bc2564827796e0796144f8104581acdcbcd7721615ad376f70baf87d",
+				ChainID:        "SN_SEPOLIA",
 				FnCall: rpc.FunctionCall{
 					Calldata: utils.TestHexArrToFelt(t, []string{
 						"0x1",
-						"0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7",
-						"0x83afd3f4caedc6eebf44246fe54e38c95e3179a5ec9ea81740eca5b482d12e",
-						"0x0",
-						"0x3",
-						"0x3",
-						"0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7",
+						"0x517567ac7026ce129c950e6e113e437aa3c83716cd61481c6bb8c5057e6923e",
+						"0xcaffbd1bd76bd7f24a3fa1d69d1b2588a86d1f9d2359b13f6a84b7e1cbd126",
+						"0x7",
+						"0x457874726163745265736f7572636546696e697368",
+						"0x5",
+						"0x5",
+						"0xb82",
 						"0x1",
-						"0x0",
+						"0x1",
+						"0x35c",
 					}),
 				},
 				TxDetails: rpc.TxDetails{
-					Nonce:   utils.TestHexToFelt(t, "0x2"),
-					MaxFee:  utils.TestHexToFelt(t, "0x574fbde6000"),
+					Nonce:   utils.TestHexToFelt(t, "0x3cf"),
+					MaxFee:  utils.TestHexToFelt(t, "0x1a6f9d0dc5952"),
 					Version: rpc.TransactionV1,
 				},
 			},
 			{
-				ExpectedHash:   utils.TestHexToFelt(t, "0x135c34f53f8b7f59efd450eb689fccd9dd4cfe7f9d9dc4d09954c5653138698"),
+				ExpectedHash:   utils.TestHexToFelt(t, "0x585419091c2f819470e30437d284b7507cb794c695fa92f801cd0a14f75483a"),
```

Comment:
Would you mind removing this test?

Reply:
Didn't find this line in the code. Hence, considered fixed.